frontend/.eslintrc.js

```diff
@@ -17,8 +17,26 @@ module.exports = {
     'vue/v-slot-style': 'off',
     'import/no-named-default': 'off',
     'import/extensions': 'off',
+    'vuejs-accessibility/alt-text': 'off',
+    'vuejs-accessibility/anchor-has-content': 'off',
+    'vuejs-accessibility/click-events-have-key-events': 'off',
+    'vuejs-accessibility/form-control-has-label': 'off',
+    'vuejs-accessibility/interactive-supports-focus': 'off',
+    'vuejs-accessibility/label-has-for': 'off',
+    'vuejs-accessibility/mouse-events-have-key-events': 'off',
+    'vue/no-reserved-component-names': 'off',
+    'vue/no-v-text-v-html-on-component': 'off',
```

Comment:
With the packages related to eslint that were updated and new defaults for recommended lint rules, such as higher requirement for accessibility in the recent years, these rules had to be explicitly added here to avoid linter errors/warnings. Alternatively we could have changed the codebase to make sure these conditions are all met but that would increase the scope and time required for this PR by a lot.

Reply:
I see, makes sense 👍🏻 could we have a separate issue for this? As a PO I'd like to track this technical debt.

Reply:
I saw that there is already an [issue](https://github.com/MetabolicAtlas/private-issues/issues/108) for this and have added a comment linking to this thread.